packages/runtime/container-runtime/src/gc/gcConfigs.ts

```diff
@@ -161,9 +166,15 @@ export function generateGCConfigs(
 		mc.config.getBoolean(gcTestModeKey) ?? createParams.gcOptions.runGCInTestMode === true;
 	// Whether we are running in tombstone mode. This is enabled by default if sweep won't run. It can be disabled
 	// via feature flags.
-	const tombstoneMode = !shouldRunSweep && mc.config.getBoolean(disableTombstoneKey) !== true;
+	const tombstoneMode = mc.config.getBoolean(disableTombstoneKey) !== true;
```

Comment:
If you're deleting `if (this.configs.tombstoneMode) {` in `runSweepPhase`, do you need this flag anymore? Just a thought. Feel free to resolve

Reply:
Feel free to disregard

Reply:
Actually I shouldn't have deleted that in this PR, added it back.